src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -257,9 +257,10 @@ const PersonalInfoList = ({ myProf, profile, createSnackbar }) => {
     createSnackbar(status.message, status.type);
   };
 
-  const graduationYear = isAlumni && !profile.PreferredClassYear.trim().isEmpty() ? (
-    <ProfileInfoListItem title={'Graduation Year:'} contentText={profile.PreferredClassYear} />
-  ) : null;
+  const graduationYear =
+    isAlumni && !profile.PreferredClassYear.trim().length === 0 ? (
```

Comment:
Since you're just trying to check whether this string has any content, we can take advantage of JavaScript's truthiness rules to simplify this. Also, we should be careful of the case where `PreferredClassYear` is null (even though that ought to be impossible, the integrity of our Profile data is unfortunately suspect).
```suggestion
    isAlumni && profile.PreferredClassYear?.trim() ? (
```

This way, we check that the person is an alumni, and that their preferred class year is not null or undefined and contains content other than whitespace.